public/openapi.yaml

```diff
@@ -526,14 +622,24 @@ paths:
                   description: V prípade použitia spoločného prístupu k schránke identifikátor pripojenia v systéme Govbox Pro
                 api_connection:
                   type: object
-                  description: Parametre pripojenia na ÚPVS
+                  description: Parametre pripojenia na FS
                   properties:
                     sub:
-                      description: V prípade pripojenia cez legacy GovBox API sa používa identifikátor subjektu.
+                      description: Identifikátor subjektu (ľubovoľná hodnota)
                       type: string
                     api_token_private_key:
-                      description: Privátny kľúč k pripojeniu na ÚPVS cez legacy GovBox API.
+                      description: Privátny kľúč k pripojeniu na FS cez legacy FS API
                       type: string
+                    settings:
+                      type: object
+                      description: Nastavenia API prepojenia
+                      properties:
+                        username:
+                          description: Používateľské meno k účtu na finančnej správe
+                          type: string
+                        password:
+                          description: Heslo k účtu na finančnej správe
+                          type: string
```

Comment:
Toto je este na zvazenie, ked chceme povolit vytvaranie `api_connection` cez API, ako vyriesit meno, heslo. Zatial len takto. @celuchmarek @jsuchal

Reply:
@luciajanikova asi nerozumiem otazke.

Reply:
Otazka je len, ci chceme/potrebujeme povolovat FS api_connection cez API. Je to trochu ina situacia v porovnani s UPVS domenou, kedze tu by sa posielali v requeste prihlasovacie udaje.